src/PhpSpreadsheet/Cell/DataValidator.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace PhpOffice\PhpSpreadsheet\Cell;
+
+use PhpOffice\PhpSpreadsheet\Calculation;
+use PhpOffice\PhpSpreadsheet\Calculation\Functions;
+
+class DataValidator
+{
+    /**
+     * Does this cell contain valid value?
+	 * 
+     * @param Cell $cell Cell to check the value
+     *
+     * @return bool
+     */
+    public function isValidCellValue(&$cell)
```

Comment:
Because there is only one thing that can be validated, the method can simply be "isValid". If you think it might not be clear enough, then we should find a better name for the class, but not the method.

Also the parameter must be type hinted with Cell, and it should not be passed by reference.